.npmignore

```diff
@@ -1 +1,14 @@
+autodeploy.sh
+.babelrc
+data/
```

Comment:
It would be easier to use a whitelist in `package.json`'s `files` than maintain this blacklist here; WDYT?